.github/workflows/cfn-publish-resource.yaml

```diff
@@ -0,0 +1,94 @@
+name: Atlas CFN Resource Publish
```

Comment:
One last question: what is the process in case the release process fails for one region? We can't restart the workflow otherwise we will release again a new version in all the regions

Reply:
@andreaangiolillo  We can point to failed region and trigger the workflow again if failed.
The failure behavior can be due to various reasons. 
eg: API fails during CFN tests. API behaves during concurrent executions e.t.c..,

As of now, we have to manually verify the in each region whether it published successfully or not.
But we can take up as an **improvement** and find a way to to know the build status in each region and echo in the workflow response.